shapely/constructive.py

```diff
@@ -926,6 +927,61 @@ def simplify(geometry, tolerance, preserve_topology=True, **kwargs):
         return lib.simplify(geometry, tolerance, **kwargs)
 
 
+@multithreading_enabled
+def polygonhull_simplify(
+    geometry, parameter, parameter_mode="area", is_outer=True, **kwargs
+):
+    """Computes a boundary-respecting hull of a polygonal geometry, with hull
+    shape determined by a target parameter specifying the fraction of the input
+    vertices or the input geometry's area retained retained in the result.
+
+    Parameters
+    ----------
+    geometry : Geometry or array_like
+    parameter : float or array_like
+        Larger values produce less concave results. A value of 1 produces
+        the convex hull; a value of 0 produces the original geometry.
+    parameter_mode : str
+        vertice - Fraction of input vertices retained, area - Ratio of simplified hull area to input area
```

Comment:
I think that it should be "vertex", not "vertice".

Reply:
Thanks for your opinion, I'll adjust the name.